lib/Migration/Version4000Date20241108004215.php

```diff
@@ -77,7 +77,8 @@ public function postSchemaChange(IOutput $output, Closure $schemaClosure, array
 		}
 		$output->advance(1);
 
-		$this->config->setAppValue(Application::APP_ID, 'providers', '');
+		// todo: maybe migrations also need to use the newer app config service?
+		$this->appConfig->setAppValueString('providers', '', true);
```

Comment:
maybe migrations also need to use the newer app config service?

Reply:
Yes, that would be good.